Sources/Jetworking/Client/ResponseHandler.swift

```diff
@@ -0,0 +1,77 @@
+import Foundation
+
+enum ResponseHandler {
+    typealias WrappedCompletion<ResponseType> = (HTTPURLResponse, Data?, Decoder, @escaping Client.RequestCompletion<ResponseType>) -> () -> Void
+    enum CompletionWrapper {
+        static func void<ResponseType>(
```

Comment:
A function should contain a verb what it is doing. Maybe we can use something like `wrapCompletionWithVoidResponse` or similar. What do you think?

Reply:
These methods are now renamed